android/README.md

```diff
@@ -1,8 +1,15 @@
 # TradingView Charting Library Android Integration Example
 
+The earliest supported version of the charting library for these examples is `v20.033`.
```

Comment:
I think we can use the major version everywhere only because 1) we're going to use stable versions here 2) we don't add breaking changes to stable version since they are released